ocs/rook.py

```diff
@@ -138,3 +140,53 @@ def _get_pod_labels(self, pod_name, namespace):
         """
 
         return self._api_client.get_labels(pod_name, namespace)
+
+    def create_cephblockpool(
+        self,
+        cephblockpool_name,
+        namespace,
+        service_cbp,
+        failureDomain,
+        size
+    ):
+        """
+        Creates cephblock pool
+
+        Args:
+            cephblockpool_name (str): name of cephblockpool
```

Comment:
A capital letter at the beginning of the sentence - fix all

Reply:
Done